client/Lykke.Service.PayInternal.Client/Api/ISupervisorApi.cs

```diff
@@ -9,11 +9,11 @@ namespace Lykke.Service.PayInternal.Client.Api
 {
     public interface ISupervisorApi
     {
-        [Get("/api/supervising/{merchantId}/{employeeId}")]
-        Task<SupervisingMerchantsResponse> GetSupervisingMerchantsAsync([Query] string merchantId, string employeeId);
+        [Get("/api/supervising/{employeeId}")]
+        Task<SupervisingMerchantsResponse> GetSupervisingMerchantsAsync(string employeeId);
 
-        [Delete("/api/supervising/{merchantId}/{employeeId}")]
-        Task DeleteSupervisingMerchantsAsync([Query] string merchantId, string employeeId);
+        [Delete("/api/supervising/{employeeId}")]
+        Task DeleteSupervisingMerchantsAsync(string employeeId);
```

Comment:
Didn't get the idea of the methods names.
This is supervisor api, so you are getting and deleting supervisor entity but not SupervisingMerchants entity.